docs/abbreviations.md

```diff
@@ -0,0 +1,4 @@
+*[GNN]: Gradient Nearest Neighbor
+*[MSN]: Most Similar Neighbor
+*[kNN]: k-Nearest Neighbor
+*[RFNN]: Random Forest Nearest Neighbor
```

Comment:
This is a cool feature of `mkdocs`. By adding this file to the `pymdownx.snippets.auto_append` in `mkdocs.yml`, the definitions will show up as tooltips anywhere in the docs that the abbreviation appears.

Reply:
That is super cool!

Reply:
For your consideration: I've used "k-nearest neighbors" in some sections of `index.md`.  I think I prefer the lower-cased "nearest" and "neighbors", but I know the acronym is kNN.  Now I'm reconsidering my decision.  Any input?

Reply:
I'm happy with the lowercase "k-nearest neighbors", despite the slight mismatch with the initials. That seems a little more commonly used, from my quick Googling. I'll make that change to abbreviations.